salvo/README.md

```diff
@@ -0,0 +1,73 @@
+# salvo
+
+This is a framework that abstracts executing multiple benchmarks of the Envoy Proxy using [NightHawk](https://github.com/envoyproxy/nighthawk).
+
+## Example Control Documents
+
+The control document defines the data needed to excute a benchmark.  At the moment the dockerized scavenging benchmark is the only one supported.  To run the benchmark, create a file with the following example contents:
```

Comment:
Maybe a small glossary that describes what we mean when we say things like "scavenging" is helpful to those new to this?

Reply:
Right!  I'll update this comment.  Somehow I am fixated on the 'scavenging' benchmark, this is really the 'fully dockerized' test.